satpy/readers/mwr_l1c.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2024 Pytroll Developers
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Reader for the Arctic Weather Satellite (AWS) MWR level-1c data.
+
+MWR = Microwaver Radiometer, onboard AWS and EPS-Sterna
+
+Sample data provided by ESA September 27, 2024.
+"""
+
+from satpy.readers.mwr_l1b import MWR_CHANNEL_NAMES, AWS_EPS_Sterna_BaseFileHandler, mask_and_scale
+
+
+class AWS_MWR_L1CFile(AWS_EPS_Sterna_BaseFileHandler):
+    """Class implementing the AWS L1c Filehandler.
+
+    This class implements the ESA Arctic Weather Satellite (AWS) Level-1b
+    NetCDF reader. It is designed to be used through the :class:`~satpy.Scene`
+    class using the :mod:`~satpy.Scene.load` method with the reader
+    ``"aws_l1c_nc"``.
+
+    """
+    def __init__(self, filename, filename_info, filetype_info, auto_maskandscale=True):
+        """Initialize the handler."""
+        super().__init__(filename, filename_info, filetype_info, auto_maskandscale)
+        self.filename_info = filename_info
+
+    @property
+    def sensor(self):
+        """Get the sensor name."""
+        return "MWR"
+
+    def get_dataset(self, dataset_id, dataset_info):
+        """Get the data."""
+        if dataset_id["name"] in MWR_CHANNEL_NAMES:
+            data_array = self._get_channel_data(dataset_id, dataset_info)
+        elif (dataset_id["name"] in ["longitude", "latitude",
+                                     "solar_azimuth", "solar_zenith",
+                                     "satellite_zenith", "satellite_azimuth"]):
```

Comment:
My suggestions in the YAML portion adds `_angle` for clarity and consistency with other readers to these
```suggestion
        elif (dataset_id["name"] in ["longitude", "latitude",
                                     "solar_azimuth_angle", "solar_zenith_angle",
                                     "satellite_zenith_angle", "satellite_azimuth_angle"]):
```